lib/protobuf/protoc/context.ex

```diff
@@ -4,32 +4,33 @@ defmodule Protobuf.Protoc.Context do
   # Plugins passed by options
   defstruct plugins: [],
 
-            ### All files scope
+  ### All files scope
```

Comment:
Please revert the changes in this file, the new formatting is not the one supported by the formatter so there is no reason in reformatting this file :)

Reply:
> Please revert the changes in this file, the new formatting is not the one supported by the formatter so there is no reason in reformatting this file :)

Formatting as in Markdown formatting or Protobuf formatting?